CODEOWNERS

```diff
@@ -0,0 +1 @@
+.github/workflows/dependency-review.yml @coveo/dev-tooling-reviewers-3
```

Comment:
Members are distributed within `@coveo/dev-tooling-reviewers-1`, `@coveo/dev-tooling-reviewers-2` and `@coveo/dev-tooling-reviewers-3`. You'll need to add the missing ones

Reply:
```suggestion
.github/workflows/dependency-review.yml @coveo/dev-tooling-reviewers-1 @coveo/dev-tooling-reviewers-2 @coveo/dev-tooling-reviewers-3
```